scripts/jest/setupEnvironment.js

```diff
@@ -44,7 +44,4 @@ if (typeof window !== 'undefined') {
       event.preventDefault();
     }
   });
-
-  // Mock for ReactART.
-  HTMLCanvasElement.prototype.getContext = () => ({});
```

Comment:
@gaearon this was added in your commit [here](https://github.com/facebook/react/pull/11956/commits/c8e879506ff2ee62c924066d153fa5844ab4c128), do you happen to remember why it is necessary? I didn't see anything throwing without it

Reply:
Pretty sure `ReactART-test` failed without it

Reply:
Passed locally, let's see if it fails in CI